client.go

```diff
@@ -44,6 +44,8 @@ type Users interface {
 	RetrieveSingleUserWithUserName(ctx context.Context, userName string, opt ...*RetrieveUserOption) (*UserResponse, error)
 	Followers(ctx context.Context, userID string, opt ...*FollowOption) (*FollowersResponse, error)
 	Following(ctx context.Context, userID string, opt ...*FollowOption) (*FollowingResponse, error)
+	PostFollowing(ctx context.Context, id string, tuid string) (*PostFollowingResponse, error)
+	UndoFollowing(ctx context.Context, suid string, tuid string) (*UndoFollowingResponse, error)
```

Comment:
I think some parameters (e.g. tuid, id) are not clearly.
This args are shown user. So, we must define arg name more meaningful.